client/my-sites/plans/current-plan/index.jsx

```diff
@@ -193,7 +195,244 @@ class CurrentPlan extends Component {
 	}
 
 	renderEcommerceTrialPage() {
-		return <div className="current-plan__ecommerce-trial-wrapper">{ this.renderMain() }</div>;
+		const { translate } = this.props;
+
+		return (
+			<>
+				<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+				<Card className="current-plan__trial-card">
+					<div className="current-plan__trial-card-content">
+						<Badge type="info">{ translate( '5 days left' ) }</Badge>
+						<p className="current-plan__card-title">
+							{ translate( 'You’re in a free trial store' ) }
+						</p>
+						<p className="current-plan__card-subtitle">
+							{ translate(
+								'Your free trial we’ll expire in 5 days. Pick a plan by December, 13th to keep your store running.'
+							) }
+						</p>
+						<Button primary>{ translate( 'Pick a plan' ) }</Button>
+					</div>
+					<div className="current-plan__trial-ilustration">
+						<img
+							width={ 180 }
+							alt="Pick a plan"
+							src="/calypso/images/plans/wpcom/plan-ecommerce-trial.png"
+						/>
+					</div>
+				</Card>
+
+				<h2 className="current-plan__section-title">What’s included in your free trial</h2>
+				<div className="current-plan__included-wrapper">
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
```

Comment:
Nit: `illustration` should have two `l`s
```suggestion
							className="current-plan__included-illustration"
```